styles/styles.css

```diff
@@ -0,0 +1,807 @@
+    display: none;
+  }
+}
+
+/* ==== Mobile M - 375px ==== */
+@media screen and (width >= 375px) {
+  .section-container {
+    padding: 2em 1em;
+  }
+
+  .section-title {
+    font-size: 1.5rem;
+  }
+
+  .hero-section {
+    height: 56vh;
+  }
+
+  /* ==== HERO SECTION ==== */
+  .hero-event {
+    font-size: 1.2rem;
+  }
+
+  /* ==== MAIN PROGRAM SECTION ==== */
+  .card-box {
+    grid-template-columns: repeat(2, 50%);
+    gap: 0.5em;
+  }
+
+  .card-box p {
+    font-size: 1.1rem;
+  }
+
+  .card-box h3 {
+    font-size: 1.4rem;
+  }
+
+  /* ==== FEATURED SPEAKERS SECTION ==== */
+  .speaker-box {
+    grid-template-columns: 34% 1fr;
+    gap: 1.2em;
+  }
+
+  .speaker-img {
+    width: 110px;
+  }
+
+  /* ==== PARTNER SECTION ==== */
+  .partner-box {
+    font-size: 1.4rem;
+    padding: 2em 0;
+  }
+
+  .speaker-infos .description {
+    font-size: 1.1rem;
+  }
+}
+
+/* ==== Tablet > 424px ==== */
+@media screen and (width > 424px) {
+  .hero-section .title {
+    font-size: 2rem;
+  }
+
+  .hero-section .subtitle {
+    font-size: 2.5rem;
+  }
+
+  .hero-section .description {
+    font-size: 1.5rem;
+  }
+
+  .program-box {
+    grid-template-columns: 34% auto;
+  }
+
+  .program-box h3 {
+    font-size: 1.5rem;
+  }
+
+  .program-box p {
+    font-size: 1.19rem;
+  }
+
+  .btn {
+    font-size: 1.2rem;
+  }
+
+  .btn-join-event {
+    letter-spacing: 1px;
+  }
+
+  .section-title {
+    font-size: 2rem;
+  }
+
+  .speaker-img {
+    width: 150px;
+  }
+
+  .speaker-box {
+    grid-template-columns: repeat(2, auto);
+  }
+
+  .speaker-infos h4 {
+    font-size: 1.5rem;
+  }
+
+  .speaker-infos .title {
+    font-size: 1.1rem;
+  }
+
+  .speaker-infos .separator {
+    width: 30px;
+  }
+
+  .speaker-infos .description {
+    font-size: 1.2rem;
+  }
+
+  .partner-box {
+    font-size: 1.7rem;
+  }
+}
+
+/* ==== Tablet > 424px ==== */
+@media screen and (width > 450px) {
+  .hero-section {
+    height: auto;
+  }
+}
+
+/* ==== Laptop > 768px ==== */
+@media screen and (width > 768px) {
+  /* ==== MENU ==== */
+  #menu-icon {
+    display: none;
+  }
+
+  .menu-box {
+    width: 100%;
+    height: auto;
+    justify-content: space-evenly;
+    align-items: center;
+    padding: 0.3em 2em 0.3em 0;
+    border-radius: 0;
+    box-shadow: 0 1px 15px -6px var(--color-dark);
+    z-index: 1000;
+  }
+
+  .menu-box .hidden-item {
+    display: block;
+  }
+
+  .menu-box .red-bordered {
+    padding: 0.3em;
+    color: var(--color-darkOrange);
+    border: 4px solid var(--color-darkOrange);
+  }
+
+  .menu-box ul {
+    display: flex;
+    align-items: center;
+    gap: 1.5em;
+  }
+
+  .menu-box ul li {
+    opacity: 1;
+    transform: translateY(0);
+    margin: 0;
+  }
+
+  .menu-box ul li a {
+    font-weight: 400;
+    font-size: 20px;
+    text-shadow: none;
+  }
+
+  .menu-box ul li a:hover,
+  .menu-box ul li a:focus {
+    transform: translateY(-3px);
+  }
+
+  .section-container {
+    max-width: 70%;
+    margin: auto;
+  }
```

Comment:
- [ ] Ouch! this is close but you seem to be missing some spacing here as shown below (highlighted in yellow) in the original design. As a frontend developer, you have to have a good eye for small details like this so please make sure to work on them.
- Original design:
  ![image](https://github.com/lily4178993/leaders-week/assets/104968236/51265134-2e03-43f3-bfec-7bfdd558c087)
- Your design:
  ![image](https://github.com/lily4178993/leaders-week/assets/104968236/2040e742-8679-44fd-bbcb-d9ce9fedd6db)